ibtk/include/ibtk/PETScMatUtilities.h

```diff
@@ -118,6 +118,25 @@ class PETScMatUtilities
 
     /*!
      * \brief Construct a parallel PETSc Mat object corresponding to the
+     * side-centered viscous operator of a side-centered velocity variable
+     * restricted to a single SAMRAI::hier::PatchLevel.
+     *
+     * \note The scaling factors of \f$ C \f$ and \f$ D \f$ variables in
+     * the PoissonSpecification object are passed separately and are denoted
+     * by \f$ \beta \f$ and \f$ \alpha \f$, respectively.
+     */
+    static void constructPatchLevelVCSCViscousOp(Mat& mat,
```

Comment:
How about SCViscousOp instead of VCSCViscousOp?

The viscous operator is sort of inherently addressing the variable coefficient case, isn't it? In the constant-coefficient case, you would just use a (vector) Laplace operator.

Reply:
I agree. However, Nishant and I are keeping VC prefix in all the routines/classes to keep track of extra code files and classes in the IBAMR repo that are specific to variable coefs solvers. If you fell strongly about removing VC prefix from this class we can do that too (We would then have to remove VC from other classes too :-( ).